app/assets/stylesheets/components/_alert.scss

```diff
@@ -49,7 +49,8 @@
     @include inline-block();
   }
 
-  .alert-valid {
+  .alert-valid,
+  .alert-notice {
```

Comment:
Where's `alert-notice` used? I don't see it elsewhere in the changes here?

Also, if you want to you could use nested syntax like:

```
.alert {
  &-valid,
  &-notice {
    … 
  }   
}
```

But that's probably overkill here.

Reply:
It's in the new feedback_controller, where we redirect to the root url with a `notice`.